core/templates/dev/head/filters/string-utility-filters/camel-case-to-hyphens.pipe.spec.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Tests for CamelCaseToHyphens filter for Oppia.
```

Comment:
Instead of making a new file can we do it such that it shows a change in the filename and change in file content? (not sure whether that's possible but I think most of the PR related to the angular project has gone in that way!)

Reply:
Didn't get. Could you please elaborate?